.travis.yml

```diff
@@ -3,14 +3,20 @@ python: 2.7
 sudo: false
 env:
   matrix:
-  - TOX_ENV=py26
-  - TOX_ENV=py27
-  - TOX_ENV=py33
-  - TOX_ENV=py34
-  - TOX_ENV=pypy
-  - TOX_ENV=docs
-  - TOX_ENV=system-tests
-  - TOX_ENV=system-tests3
+  # - TOX_ENV=py26
+  # - TOX_ENV=py27
+  # - TOX_ENV=py33
+  # - TOX_ENV=py34
+  # - TOX_ENV=pypy
+  # - TOX_ENV=docs
+  # - TOX_ENV=system-tests
+  # - TOX_ENV=system-tests3
+  - TOX_ENV=gae
+  global:
+  - GAE_PYTHONPATH=$HOME/google_appengine
+cache:
+  directories:
+  - $HOME/google_appengine
```

Comment:
I think we should use `${HOME}/.cache/google_appengine`. It's what I've seen be used most commonly as the root directory (and gives a clear indication of purpose).